### ISSUE TITLE
Radar doesn't update rules when using a self-hosted instance

### ISSUE THREAD

**Issue by echoriiku** [DIYgod/RSSHub-Radar#1038]
The Rules are not updated (at all) whe using a self-hosted instance. In the instance logs I get:
```sh
info: <-- GET /api/radar/rules
error: Error in /api/radar/rules: RejectError: Authentication failed. Access denied.
info: --> GET /api/radar/rules 403 1ms
```
I have correctly set up the Access Key/Code in Radar.

**Comment by bestlzk** [comment]
Encountered the same problem. After configuring the key, the key will not be actually used to initiate a request
![screenshot-20241012-111904](https://github.com/user-attachments/assets/0e5d1daf-635c-4160-ba6c-f018a828728f)

**Comment by TonyRL** [comment]
Fixed in #1048
